main.tf

```diff
@@ -99,6 +99,11 @@ resource "kubernetes_deployment" "wandb" {
             value = var.oidc_client_id
           }
 
+          env {
+            name  = "OIDC_SECRET"
+            value = var.oidc_secret
+          }
+
```

Comment:
This needs to be a secret, it shouldn't be exposed as a regular env var.  See the `MYSQL` case.  `BUCKET` should also be a secret as a customer might be running `Minio` and give us the auth in the `BUCKET` connection string.  `REDIS` can also have credentials in it's connection string.  Maybe we file a ticket?